src/main/kotlin/database/models/Player.kt

```diff
@@ -115,6 +115,7 @@ data class PlayerStats(
     val weaponDeaths: MutableMap<String, Int> = mutableMapOf(),
     val killstreaks: MutableMap<Int, Int> = mutableMapOf(),
     val killstreaksEnded: MutableMap<Int, Int> = mutableMapOf(),
+    val achievements: MutableList<String> = mutableListOf()
```

Comment:
I think it would be cool to store the date/time each achievement was reached. It's not strictly necessary though; up to you.